<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="fileChooseWindow">
     <widget class="QLineEdit" name="filePathLineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>470</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseImageButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextWindowButton1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>520</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 141, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="imagePreview">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>62</y>
        <width>431</width>
        <height>391</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(173, 173, 173)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="methodChooseWindow">
     <widget class="QPushButton" name="nextWindowButton2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>520</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 141, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousWindowButton1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>520</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 141, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>320</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>BT.601</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lume (BT.709)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>380</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Algorithmic</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Formula used by GIMP or Photoshop</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>671</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>Please choose grayscale filter method:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resultWindow">
     <widget class="QPushButton" name="previousWindowButton2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>520</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 141, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadButton1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>520</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadButton2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>520</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLabel" name="asmLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Assembler</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="cLabel">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="asmTimeLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>480</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cTimeLabel">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>480</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="asmTime">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>480</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cTime">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>480</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="asmPreview">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>371</width>
        <height>411</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(173, 173, 173)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cPreview">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>371</width>
        <height>411</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(173, 173, 173)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
